var/spack/repos/spack_repo/builtin/packages/py_torch/package.py

```diff
@@ -304,10 +307,19 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         depends_on("rocrand")
         depends_on("hipsparse")
         depends_on("hipfft")
+        depends_on("hiprand")
+        depends_on("hipsolver")
+        depends_on("rocm-core")
         depends_on("rocfft")
         depends_on("rocblas")
         depends_on("miopen-hip")
         depends_on("rocminfo")
+        depends_on("aotriton@0.8.1b", when="@2.5:2.6")
+        depends_on("aotriton@0.9.1b", when="@2.7")
+        depends_on("aotriton@0.9.1b", when="@main")
+        depends_on("composable-kernel@:6.3.2", when="@2.5.0:2.5.2")
+        depends_on("composable-kernel@6.3.2:", when="@2.6:")
```

Comment:
Just trying to understand this. PyTorch 2.4 and older do not require ck. PyTorch 2.5 requires up to and including 6.3.2 (oddly specific number). Python 2.6+ require 6.3.2+. Is that correct?